src/api/supabase/queries/languageSupport.ts

```diff
@@ -0,0 +1,30 @@
+import { LanguageSupport, CaseListing } from '@/types/schema';
+import supabase from '../createClient';
+
+/**
+ * Fetches all non-case-specific language support entries from language-support database
+ * @returns a Promise of all non-case-specific language support objects
+ */
+export async function getAllLanguageSupport(): Promise<LanguageSupport[]> {
+  const { data, error } = await supabase.from('language-support').select('*');
+  if (error) {
+    throw new Error(`Error reading language support: ${error.message}`);
+  }
+  return data;
+}
+
+/**
+ * Fetches all case interpetation entries from cases database + assigns their listing_type assigned as "Case Interpretation"
+ * @returns a Promise of all case interpretation objects
+ */
+export async function getCaseInterpretations(): Promise<CaseListing[]> {
+  const { data, error } = await supabase.rpc('getCasesData');
+  if (error) throw new Error(`Error fetching all cases: ${error.message}`);
+  const modifiedData = data
+    .filter((c: CaseListing) => c.needs_interpreter === true)
+    .map((caseInterpretation: CaseListing) => ({
+      ...caseInterpretation,
+      listing_type: 'Case Interpretation',
```

Comment:
let's set `listing_type` to 'Case' so that it remains consistent with the `CaseListing` schema. See [this comment](https://github.com/calblueprint/immigration-justice-project/pull/66#discussion_r1520976926) for more info